isobus/src/can_transport_protocol.cpp

```diff
@@ -11,860 +11,1004 @@
-			                                                        std::static_pointer_cast<InternalControlFunction>(session->sessionMessage.get_source_control_function()),
-			                                                        nullptr,
-			                                                        CANIdentifier::CANPriority::PriorityDefault6);
+			case REQUEST_TO_SEND_MULTIPLEXOR:
+			{
+				if (message.is_destination_global())
+				{
+					CANStackLogger::warn("[TP]: Received a Request to Send (RTS) message with a global destination, ignoring");
+				}
+				else
+				{
+					const auto totalMessageSize = message.get_uint16_at(1);
+					const auto totalNumberOfPackets = message.get_uint8_at(3);
+					const auto clearToSendPacketMax = message.get_uint8_at(4);
+					process_request_to_send(message.get_source_control_function(),
+					                        message.get_destination_control_function(),
+					                        pgn,
+					                        totalMessageSize,
+					                        totalNumberOfPackets,
+					                        clearToSendPacketMax);
+				}
+			}
+			break;
+
+			case CLEAR_TO_SEND_MULTIPLEXOR:
+			{
+				if (message.is_destination_global())
+				{
+					CANStackLogger::warn("[TP]: Received a Clear to Send (CTS) message with a global destination, ignoring");
+				}
+				else
+				{
+					const auto packetsToBeSent = message.get_uint8_at(1);
+					const auto nextPacketNumber = message.get_uint8_at(2);
+					process_clear_to_send(message.get_source_control_function(),
+					                      message.get_destination_control_function(),
+					                      pgn,
+					                      packetsToBeSent,
+					                      nextPacketNumber);
+				}
+			}
+			break;
+
+			case END_OF_MESSAGE_ACKNOWLEDGE_MULTIPLEXOR:
+			{
+				if (message.is_destination_global())
+				{
+					CANStackLogger::warn("[TP]: Received an End of Message Acknowledge message with a global destination, ignoring");
+				}
+				else
+				{
+					process_end_of_session_acknowledgement(message.get_source_control_function(),
+					                                       message.get_destination_control_function(),
+					                                       pgn);
+				}
+			}
+			break;
+
+			case CONNECTION_ABORT_MULTIPLEXOR:
+			{
+				if (message.is_destination_global())
+				{
+					CANStackLogger::warn("[TP]: Received an Abort message with a global destination, ignoring");
+				}
+				else
+				{
+					const auto reason = static_cast<ConnectionAbortReason>(message.get_uint8_at(1));
+					process_abort(message.get_source_control_function(),
+					              message.get_destination_control_function(),
+					              pgn,
+					              reason);
+				}
+			}
+			break;
+
+			default:
+			{
+				CANStackLogger::CAN_stack_log(CANStackLogger::LoggingLevel::Warning, "[TP]: Bad Mux in Transport Protocol Connection Management message");
```

Comment:
Since we're changing a bunch in this file anyways we might as well consider replacing some of these with the shorter ones.
```suggestion
				CANStackLogger::warn("[TP]: Bad Mux in Transport Protocol Connection Management message");
```